### ISSUE TITLE
[Bug]: Sync process is triggered every second on version 3.16.3

### ISSUE THREAD

**Issue by muchachagrande** [nextcloud/desktop#6701]
### ⚠️ Before submitting, please verify the following: ⚠️

- [X] This is a **bug**, not a question or a configuration issue.
- [X] This issue is **not** already reported on Github (I've searched it).
- [X] Nextcloud Server and Desktop Client are **up to date**. See [Server Maintenance and Release Schedule](https://github.com/nextcloud/server/wiki/Maintenance-and-Release-Schedule) and [Desktop Releases](https://nextcloud.com/install/#install-clients) for supported versions.
- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/)

### Bug description

The sync process is triggered on each second and it immediately shows "All synced!" producing the logs folder to flood of sync logs.
---

### Steps to reproduce

1. Open the main dialog
2. Observe the sync icon on the top flash between syncing and synced states
3. Observe de ~/.config/nextcloud/logs directory flooding of sync logs
...

### Expected behavior

Once all files are synced the sync icon should stay quiet.
...

### Which files are affected by this bug

No files affected

### Operating system

Linux

### Which version of the operating system you are running.

Ubuntu 24.04

### Package

Appimage

### Nextcloud Server version

31.0.4

### Nextcloud Desktop Client version

3.16.3

### Is this bug present after an update or on a fresh install?

Updating from previous versions and fresh installing

### Are you using the Nextcloud Server Encryption module?

Encryption is Disabled

### Are you using an external user-backend?

- [X] Default internal user-backend
- [ ] LDAP/ Active Directory
- [ ] SSO - SAML
- [ ] Other

### Nextcloud Server logs

```shell
No server logs
```

### Additional info

I had to revert to 3.12.3 client. The logs folder has flooded very fast.

**Comment by weinbrenner** [comment]
Seems to be the same for me here. Nextcloud Desktop 3.13.0, Linux AppImage, Ubuntu 22.04
Process(es) is running a lot of the time at 100%, Laptop is getting loud and battery is draining fast!

Was never the case in Versions 3.12.x and older.

**Comment by weinbrenner** [comment]
# ls -l .config/Nextcloud/logs/ | wc -l
12183

less than a minute later:
# ll .config/Nextcloud/logs/ | wc -l
12198

each logfile is round about 2MB and all the 12 thousend files are from within 2 days ....

**Comment by floss4good** [comment]
I'm also on Ubuntu 22.04.4 but I don't use the AppImage; I use the app provided through [ppa:nextcloud-devs/client](https://launchpad.net/~nextcloud-devs/+archive/ubuntu/client) repository.
As for the server, the most recent version is 27.1.5 (I have multiple accounts added in the desktop client).
I updated earlier to 3.13.0 and it seems I'm not affected by this issue.
Therefore, as an alternative workaround you can try to install the app through the mentioned PPA.

**Comment by weinbrenner** [comment]
> I'm also on Ubuntu 22.04.4 but I don't use the AppImage; I use the app provided through [ppa:nextcloud-devs/client](https://launchpad.net/~nextcloud-devs/+archive/ubuntu/client) repository. As for the server, the most recent version is 27.1.5 (I have multiple accounts added in the desktop client). I updated earlier to 3.13.0 and it seems I'm not affected by this issue. Therefore, as an alternative workaround you can try to install the app through the mentioned PPA.

Good idea, thanks - but it didn't help, behavior is still the same. My server version is 27.1.8.
Maybe it depends on how many files you are syncing - in my case the total is around 50GB. (But almost all files should already be in sync, probably only a few files which I worked on since upgrading to 3.13.0

**Comment by muchachagrande** [comment]
I'm syncing a few text files to keep my notes app in sync with the NC server.

**Comment by dom6770** [comment]
I have the same issue on Windows 11 with 3.13.0 and NC 29.0.1:

https://github.com/nextcloud/desktop/assets/11932291/cdf48d19-baeb-4a3f-8755-c1502e8910d9

After two hours now I have 2.729 items with a size of 676 MiB.

**Comment by weinbrenner** [comment]
another ticket reports (maybe) the same problem: #6771

**Comment by muchachagrande** [comment]
I've just tested 3.13.1 and this problem is still present.

**Comment by weinbrenner** [comment]
I've just tested 3.13.2 and this problem is still present.

**Comment by jaymerts** [comment]
Here I have this running forever:
![image](https://github.com/user-attachments/assets/1bb813f0-cb5e-4480-8a09-6ddd019c17fb)

**Comment by muchachagrande** [comment]
I've tested 3.14.0 and this problem is present.
The sync process is constantly triggering once per second.

**Comment by muchachagrande** [comment]
I've tested 3.14.1 and is also affected by this problem.
Checking for changes every second.

**Comment by muchachagrande** [comment]
Problem still present in 3.14.3

**Comment by ulbo-sudo** [comment]
I'm having the same issue with Appimage on Linux Mint 21.3 Cinnamon. 

The sync process is triggering about every 3 seconds and the disk gets flooded with logfiles.

**Comment by ulbo-sudo** [comment]
Problem still present in 3.15.0

**Comment by muchachagrande** [comment]
I'm now on Ubuntu 24.04 upgraded from 22.04.
Actual desktop version is 3.16.2.

This issue is still present.

I'll try installing from scratch instead of updating. May be is something related to permissions... I don't know...

**Comment by muchachagrande** [comment]
I've an update on this issue. I didn't test the rest of 3.16.x series and jumped to test 3.17.0 RC series.

Yesterday I've installed 3.17.0 RC2 on two PCs with Ubuntu 24.04 and this issue didn't happen on neither one.

I'll be testing this and update and test the final version when it comes. If the issue doesn't happen then, I'll close this bug.

**Comment by muchachagrande** [comment]
I've tested 3.17.0 RC3 and now I'm testing 3.17.0 final.... It is working fine. The sync triggers only when intended so I'm closing this issue.
